src/main/webapp/auth.js

```diff
@@ -79,13 +79,10 @@ function handleAuthenticationState() {
     // Populate information panels at top of dashboard
     populateGmail();
     populateTasks();
-    populateCalendar();
+    // populateCalendar();
```

Comment:
Um, did you intend to comment this out? It seems unrelated to the Go panel.

Reply:
Yeah sorry about that, I wanted to screen capture the deployed site and this function writes a bunch of holidays from calendar behind the Go panel. However, we just decided it's possible to merge everything and then do our screen capture off the final product.

Reply:
`RESOLVED`